### DISCUSSION TITLE
Control the frequency of the Time codes

### DISCUSSION THREAD

**Discussion by MaxHaller91** [Dadangdut33/Speech-Translate#7] [Ideas]
Thanks so much for putting this program out here. Is there any adjustment possible so i get timecodes for every second? or even every word?

**Comment by Dadangdut33** [comment]
hmm it might be possible actually for timecodes of every word, someone actually made that into a [library](https://github.com/Jeronymous/whisper-timestamped) as explained in [this discussion](https://github.com/openai/whisper/discussions/3). If you want i could add it
